functions/Get-DbaComputerSystem.ps1

```diff
@@ -55,6 +58,7 @@ function Get-DbaComputerSystem {
         [DbaInstanceParameter[]]$ComputerName = $env:COMPUTERNAME,
         [PSCredential]$Credential,
         [switch]$IncludeAws,
+        [int]$TimeoutSec = 5,
```

Comment:
Why does this need to be a parameter? As well, we had it set to 15 previously, if the parameter is to stay in place it needs to be set to 15 seconds.

Reply:
Talking with Chrissy this needs to be removed and implemented as a module configuration. 

However from my point of view I see no reason for it. The site you are hitting is a local site in AWS servers only. I believe we can simply set it to a value and leave it.

Reply:
Sounds good to me. thanks!